pkg/cloudprovider/aws/capacity.go

```diff
@@ -97,7 +98,15 @@ func (c *Capacity) GetInstanceTypes(ctx context.Context) ([]string, error) {
 }
 
 func (c *Capacity) GetZones(ctx context.Context) ([]string, error) {
-	return c.vpcProvider.GetZones(ctx, c.spec.Cluster.Name)
+	azs, err := c.vpcProvider.GetAllAvailabilityZones(ctx)
```

Comment:
Maybe call this `GetAllZones` (or even just `GetZones`) for consistency (since this method is called `GetZones`?)

Reply:
good point 👍  fixed